cli/test/cli/index-test.js

```diff
@@ -80,81 +80,4 @@ describe('CLI Tests', function() {
       assert.equal(ret.status, 1);
     });
   });
-
-  describe('print-config', () => {
-    describe('fraggle rock', () => {
-      it('should print the overridden config and exit immediately after', () => {
-        const flags = [
-          '--print-config', '-A',
-          '--output', 'json',
-          '--only-audits', 'metrics',
-        ];
-        const ret = spawnSync('node', [indexPath, ...flags], {encoding: 'utf8'});
-
-        const config = JSON.parse(ret.stdout);
-        assert.strictEqual(config.settings.output[0], 'json');
-        assert.strictEqual(config.settings.auditMode, true);
-        // FR config doesn't exclude full-page-screenshot with --only-audits
-        assert.strictEqual(config.audits.length, 2);
-
-        expect(config).toMatchSnapshot();
-      });
-    });
-
-    it('should print the overridden config and exit immediately after', () => {
-      const flags = [
-        '--print-config', '-A',
-        '--output', 'json',
-        '--only-audits', 'charset',
-        '--legacy-navigation',
-      ];
-      const ret = spawnSync('node', [indexPath, ...flags], {encoding: 'utf8'});
-
-      const config = JSON.parse(ret.stdout);
-      assert.strictEqual(config.settings.output[0], 'json');
-      assert.strictEqual(config.settings.auditMode, true);
-      assert.strictEqual(config.audits.length, 1);
-
-      expect(config).toMatchSnapshot();
-    });
-  });
-
-  describe('preset', () => {
-    it('desktop should set appropriate config', () => {
```

Comment:
would be good to still test this. 
maybe a mock for the core/ API and check the config?

Reply:
It's tested in bin-test.js

Reply:
ah.. the experimental preset. true